### DISCUSSION TITLE
[IMPLEMENTED] Select multiple books in file browser

### DISCUSSION THREAD

**Discussion by FajnB** [koreader/koreader#8405] [Ideas]
Hi, I had an idea, that it would be nice to implement a option to select multiple items/books in file browser.

_`Hit me, if I missed this feature, but so far I was not able to find any way to select multiple books at once.`_

If implemented it could be relatively easy to for example move books from folder to folder without using native e-readers file browser, as koreader is pretty much faster than my whole e-readers OS :D, and if someone is moving from one reader app to koreader to would make it easy for them to just bulk-mark many books as done reading at onece.

It is just an idea but it could be a nice quality of life feature. 😉

**Comment by Frenzie** [comment]
It'd be nice to have, but I don't think anybody's currently very interested in spending time on that. :-)

**Comment by poire-z** [comment]
Just a wish if this gets worked on: as we're mostly an end point at using files, in 99% of the actions, we deal with the current book, or an added new book (at least, _I_ do only that). So, for such case, I  would prefer to not have another step before I can proceed (I don't want to long-press on a file, and have it checked, and have to hit something else to say I'm done with my selection of one). May be a very-long-press could toggle this new mode of multi selection.

**Comment by clach04** [comment]
Implementation in https://github.com/koreader/koreader/pull/9571. Available in build/release 2023.1.
Recommend closing.

**Reply by NiLuJe** [reply]
In another great example of "GH discussions are awesome", maintainers can't actually flag a post as "resolved", apparently...

The best I can do is lock it, which is stupid, but, oh, well.

**Reply by Frenzie** [reply]
I think the OP might be able to select this as an "answer" or something?

**Reply by NiLuJe** [reply]
Yup, but I'm baffled by the fact that *we* can't do that ourselves :s.
